http/examples/get/client.R

```diff
@@ -0,0 +1,24 @@
+library(httr)
+library(arrow)
+library(tictoc)
+
+url <- 'http://localhost:8000'
+
+tic()
+
+response <- GET(url)
+buffer <- content(response, "raw")
+reader <- RecordBatchStreamReader$create(buffer)
+table <- reader$read_table()
+
+# or:
+#batches <- reader$batches()
+# but this is very slow
```

Comment:
@paleolimbot I couldn't find a fast way to convert the IPC stream into a list of record batches in R. Any ideas?